settings.py

```diff
@@ -1680,9 +1674,7 @@ def _determine_couch_databases(couch_databases):
 }
 
 # These are custom templates which can wrap default the sms/chat.html template
-CUSTOM_CHAT_TEMPLATES = {
-    "FRI": "fri/chat.html",
-}
```

Comment:
@gcapalbo it didn't look like this was overrideable in ansible, so I think this is only used by FRI.  There's also a lot of code associated with using and updating this, should that also be all removed?

Reply:
i think for now we'd want to just remove the entry in this dictionary, but not the setting itself. but yes, in general, if product agrees we might be able to remove the custom chat template feature and some other sms chat-related features that were built only for this project